src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -274,6 +274,13 @@ def test_nav_links_webextensions(self):
 class TestEditBasicListed(BaseTestEditBasic):
     __test__ = True
 
+    def test_edit_page_not_editable(self):
+        # The /edit page is the entry point for the individual edit sections,
+        # and should never display the actual forms, so it should always pass
+        # editable=False to the templates it renders.
```

Comment:
Let's add a link to the github-issue so that people understand that this caused problems?